spanner/ddl/ast.go

```diff
@@ -328,45 +328,55 @@ func NewSchema() Schema {
 	return make(map[string]CreateTable)
 }
 
+// Tables are ordered in alphabetical order with one exception: interleaved
+// tables appear after the definition of their parent table.
+func OrderTable(s Schema) []string {
```

Comment:
1. Placing this method in mapping.go causes circular dependency of mapping.go and ast.go because Schema is defined in ast.go.
2. spSchema is a map therefore unable to sort and place there